### DISCUSSION TITLE
Could you please share what tools to create 2020  RubyKaigi talk?

### DISCUSSION THREAD

**Discussion by zw963** [socketry/async#123] [General]
Hi, Samuel, because i rarely use twitter, so ask question here.

I saw your's several youtube videos, one of is [[EN] Don't Wait For Me! Scalable Concurrency for Ruby 3! / Samuel Williams @ioquatix](https://www.youtube.com/watch?v=Y29SSOS4UOc)

I am so surprise what tools to use create this awesome video talk,  could you please share this?

**Comment by ioquatix** [comment]
Hi, I use Apple Keynote, but after creating presentations for several years I gained a lot of experience with how to make good animations. Apple Keynote has a feature called "Magic Move" which tweens between objects on different slides, and you can make good animations using this transition, but you sometimes need to "cheat" and add intermediate slides or hidden objects to get the right animations.

I worked out that about each minute of a presentation takes about 8 hours of work. So, it's also just a huge amount of effort, practicing, tweaking, etc. A 30 minute presentation takes about 1-2 months of part time work at least. Recording and voice over takes about 10 minutes for every 1 minute of audio, so it would usually take between 5-10 hours to do voice-overs.

What I'm trying to say, is that there is no magic answer - it's just a huge amount of work. Of course you get better at it the more you do it, and also Keynote is pretty decent (but also has some frustrating limitations). But basically, it's just a lot of time consuming work.

In order to help you out, I'll try to upload all my presentations which I've done in the past several years to GitHub, but it won't be for a few more days when I have some free time: https://github.com/ioquatix/presentations

**Reply by ioquatix** [reply]
@zw963 Okay, I've uploaded all the presentations that I can do so easily (some have embedded vide that is huge). Feel free to check them out and ask any questions about how to do the animations. The specific one you are after is [Don't Wait For Me! Scalable Concurrency for Ruby 3! Keynote](https://github.com/ioquatix/presentations/blob/master/2020/Don't%20Wait%20For%20Me%20(Ruby%20Kaigi%20Takeout%202020).key)

**Reply by zw963** [reply]
Thank you very much, though i use arch linux,  i guess there is not possible to use keynote directly on my laptop, but i will borrow a mac to try this.

**Reply by ioquatix** [reply]
I also use Arch Linux for everything development related (nothing beats Linux for development) but for design stuff nothing beats a Mac. I actually also do post processing of video files using Final Cut Pro (audio levels, Compressor for exporting, etc). If you are planning to do presentations, I can highly recommend a MacBook or MacBook Pro (especially the new Apple Silicone hardware which is simply amazing).

**Reply by ioquatix** [reply]
Also for voice recording I mostly use my https://www.rode.com/microphones/studio/nt1-ai1kit which is pretty decent. I've had few connectivity issues (maybe cable or USB host related). The microphone itself could stand to be a little bit more sensitive, or maybe I'm just not that good at using it. I also found that it's really important to ground it properly if you are using a computer with floating ground voltage (can cause all sorts of quality issues). I may investigate microphones in more detail in the future. It's really important but honestly I'm not that experienced with them, I probably need to seek some advice from more experienced audio technicians.
